test2_2/main.c

```diff
@@ -0,0 +1,81 @@
+#include <stdio.h>
+#include "list.h"
+
+void showMenu() {
```

Comment:
```suggestion
void showCommands() { \\ or showHelpMessage
```
Разве мы меню пользователю показываем?